README.md

```diff
@@ -150,7 +150,10 @@ Complete documentation for these methods is coming with: https://github.com/ipfs
 
 > `ipfs.util.addFromFs(path, option, callback)`
 
-Reads a file from `path` on the filesystem  and adds it to IPFS. If `path` is a directory, use option `{ recursive: true }` to add the directory and all its sub-directories. To exclude fileglobs from the directory, use option `{ ignore: ['ignore/this/folder/**', 'and/this/file'] }`.
+Reads a file or folder from `path` on the filesystem  and adds it to IPFS. Options:
+- **recursive**: If `path` is a directory, use option `{ recursive: true }` to add the directory and all its sub-directories.
+  - **ignore**: To exclude fileglobs from the directory, use option `{ ignore: ['ignore/this/folder/**', 'and/this/file'] }`.
+  - **dot**: Dotfiles (files or folders starting with a `.`, for example, `.git/`) are not included by default. To add them, use the option `{ dot: true }`. 
```

Comment:
`go-ipfs` calls these 'hidden', could you change this to 'hidden' for the sake of consistency?